developer/src/common/include/kmn_compiler_errors.h

```diff
@@ -277,6 +277,14 @@ namespace KmnCompilerMessages {
 
     ERROR_TextBeforeOrAfterNulInOutput =                SevError | 0x0B3,
 
+    ERROR_NameMustBeAtLeastOneCharLong =                SevError | 0x0B4,
+    ERROR_NameMustBeAtMostNCharsLong =                  SevError | 0x0B5,
+    ERROR_NameContainsInvalidCharacter =                SevError | 0x0B6,
+    ERROR_NameMustNotContainSpaces =                    SevError | 0x0B7,
+    ERROR_NameMustNotContainComma =                     SevError | 0x0B8,
+    ERROR_NameMustNotContainParentheses =               SevError | 0x0B9,
+    ERROR_NameMustNotContainSquareBrackets =            SevError | 0x0BA,
```

Comment:
curly braces allowed though, huh?

Reply:
Curly braces are not recognized characters in Kmn, whereas square brackets are.